src/containers/Home/HomeWidgets/NotificationWidget.js

```diff
@@ -0,0 +1,269 @@
+                    borderRadius: 10,
+                    backgroundColor: '#F1F1F1'
+                }}
+                mode="elevated"
+                elevation={5}>
+                <Card.Content>
+                    <View
+                        style={{
+                            display: 'flex',
+                            flexDirection: 'row',
+                            justifyContent: 'space-between',
+                            width: width - 30,
+                            height: 10,
+                            alignItems: 'center',
+                            marginBottom: collapsed ? 0 : 15,
+                        }}>
+                        <View style={{
+                            display: 'flex',
+                            flexDirection: 'row',
+                            alignItems: 'center',
+                        }}>
+                            <View
+                                style={{
+                                    display: 'flex',
+                                    flexDirection: 'row',
+                                    alignItems: 'center',
+                                    backgroundColor: '#56B73E',  // Set the background color to green
+                                    borderRadius: 5,  // Set the border radius for rounded corners
+                                    height: 24,  // Set the height
+                                    width: 55,  // Set the width
+                                    marginRight: 2
+                                }}>
+                                <IconButton
+                                    icon="bell"
+                                    color="white"
+                                    size={15}
+                                    style={{ marginRight: 3 }}
+                                />
+                                <View style={{ justifyContent: 'center', alignItems: 'center', borderColor: 'white', borderWidth: 1, borderRadius: 15, width: 16, height: 16 }}>
+                                    <Text style={{ color: "white", textAlign: 'center', fontSize: 10 }}>
+                                        {traynotifications.length}
+                                    </Text>
+                                </View>
+                            </View>
+                            {collapsed && traynotifications.map((notification) =>
+                                notification.icon)}
+                        </View>
+                        <View style={{
+                            display: 'flex',
+                            flexDirection: 'row',
+
+                        }}>
+                            <TouchableOpacity
+                                onPress={() => { dispatchClearNotifications(); }}>
+                                {!collapsed && <Paragraph style={{ fontSize: 12, textDecorationLine: 'underline', marginVertical: 17 }}>
+                                    clear all
```

Comment:
"Clear all" should have an "are you sure message", as well as clearing individual notifs. Later this maybe should be optional based on notif type or data but for now I think its ok to add an "are you sure" more generally to all.